### PR TITLE
[ADD] migration for project_mrp

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1: It's a good habit to create the openupgrade_analysis_work file.

File: addons/project_mrp/migrations/6.1.1.0/post-migrate.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+
+def migrate(cr, version):
```
Comment (StefanRijnhart) [inline addons/project_mrp/migrations/6.1.1.0/post-migrate.py:25]: No decorator?

Comment (hbrunn) [inline addons/project_mrp/migrations/6.1.1.0/post-migrate.py:25]: @StefanRijnhart yes, better to have one

**Comment by pedrobaeza** [comment]
Can you add the analysis work file?

**Comment by hbrunn** [comment]
@pedrobaeza thanks for the reminder

**Comment by pedrobaeza** [comment]
Can anyone add a second review for this to merge it?

**Comment by StefanRijnhart** [comment]
:+1:, thanks

**Pull request merged**